xml-binding/4.0/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Jakarta XML Binding 4.0 (under development)"
+date: 2021-04-15
+summary: "Release for Jakarta EE 10"
+---
+The Jakarta XML Binding provides an API and tools that automate the mapping
+between XML documents and Java objects.
+
+The goal of this release is to:
+
+* provide features requested by the community
+* implement requirements by other specification included in Jakarta EE 10
```

Comment:
I would like to see an explicit statement regarding the required major version update.  I know we've discussed this on other Plan Reviews, but it would be good to explain the specific reason in this JAXB update.  Thanks!

Reply:
@kwsutter Dropping support for JAXB 1 related APIs/functionality (ie validation) as well as dropping explicit dependency on JavaBeans Framework are both removals, thus backward incompatible changes requiring major version change - both are explicitly mentioned (and both also fall under requirements from other specs, in this case xml-ws). Or do youmean sth else?

Reply:
@lukasj Those are the type of changes I was referring to.  I was just looking for something more explicit in the text of the release plans to indicate that.  For example, something along these lines:

- drop explicit dependency on JavaBeans (Has transitive relationship to any dependent specifications. Removal of JavaBeans is a breaking API change.)
- editorial updates, clarifications and removal of JAXB 1 from the specification (May have transitive relationship to any dependent specifications. Removal of JAXB 1 is a breaking API change.)

Just something to allow the casual reader to understand why a major version upgrade was necessary.  I know it's very natural for you and others more intimate with JAXB, but for others, it helps identify the reasons for the major version update.  Thanks!

Reply:
is it allowed to change the PR while the ballot is in progress? since it seems to be required to explicitly list them now in the plan, does it mean that updating the same entry in the release record - probably before release review - is not/won't be required to avoid duplications or is the expectation that both should be kept in sync?

Reply:
In this case, you are just further clarifying your intent of the Release Plan.  You're not modifying what was going to be included.  You are just explaining that these proposed changes are the reason behind the major version update.  That is an okay update to make in the middle of a ballot.  Thanks.